### ISSUE TITLE
Distributed PDF reference manual doesn't use calligraphic font

### ISSUE THREAD

**Issue by akr** [coq/coq#9429]
#### Version

https://github.com/coq/coq/releases/download/V8.9.0/coq-8.9.0-reference-manual.pdf

#### Operating system

% lsb_release -a
No LSB modules are available.
Distributor ID:	Debian
Description:	Debian GNU/Linux 9.7 (stretch)
Release:	9.7
Codename:	stretch

I use evince 3.22.1-3+deb9u1 to view the PDF file.

#### Description of the problem

I found the distributed PDF reference manual doesn't use a calligraphic font.
I guess the manual is formatted in a environment which lacks a font.

For example, there is a following paragraph in p.103 of coq-8.9.0-reference-manual.pdf : 

![no-calligraphic-font](https://user-images.githubusercontent.com/10538/51901507-2ff72400-23fb-11e9-9baa-8e0cb90d25f7.png)

The last two line describes the difference of two kind of "S".
But two "S" uses same font and they are not distinguishable.
The latter "S" should use calligraphic font.

The corresponding source in cic.rst is follows:

```
These inductive definitions, together with global assumptions and
global definitions, then form the global environment. Additionally,
for any :math:`p` there always exists :math:`Γ_P =[a_1 :A_1 ;…;a_p :A_p ]` such that
each :math:`T` in :math:`(t:T)∈Γ_I \cup Γ_C` can be written as: :math:`∀Γ_P , T'` where :math:`Γ_P` is
called the *context of parameters*. Furthermore, we must have that
each :math:`T` in :math:`(t:T)∈Γ_I` can be written as: :math:`∀Γ_P,∀Γ_{\mathit{Arr}(t)}, S` where
:math:`Γ_{\mathit{Arr}(t)}` is called the *Arity* of the inductive type t and :math:`S` is called
the sort of the inductive type t (not to be confused with :math:`\Sort` which is the set of sorts).
```

The latter "S" uses "\Sort" command defined in refman-preamble.sty:

```
\newcommand{\Sort}{\cal S}
```

\Sort command uses \cal command.
So, it should be formatted in calligraphic font.

Actually, it is formatted in calligraphic font in my environment when I built the reference manual from the source as : 

![uses-calligraphic-font](https://user-images.githubusercontent.com/10538/51901941-2e7a2b80-23fc-11e9-907a-31e132aa7c11.png)

So, I guess the distributed reference manual is built on a environment which lacks a font.

**Comment by SkySkimmer** [comment]
See https://github.com/coq/coq/pull/9435
It's not about font availability, it's because `\cal` doesn't work.

**Comment by Zimmi48** [comment]
While #9435 should ensure there is no such problem in future version, we can also fix the file that is currently distributed for 8.9.0. @akr Could you attach this file to this issue since you have built it already and your version looks better?

**Comment by akr** [comment]
Thank you.

I downloaded 
https://gitlab.com/SkySkimmer/coq/-/jobs/152960546/artifacts/raw/_install_ci/share/doc/coq/sphinx/latex/CoqRefMan.pdf?inline=false
and I took a screenshot:

![mathcal-calligraphic-s](https://user-images.githubusercontent.com/10538/51946914-6ae27180-2466-11e9-9907-3cbc9db8ad1f.png)

The latter "S" is correctly formatted in a calligraphic font.
So, it's fine to use the file.

**Comment by Zimmi48** [comment]
> So, it's fine to use the file.

It's not actually a copy for Coq 8.9.0.

**Comment by akr** [comment]
I understand.

Unfortunately my PDF file is not appropriate because
I have no xindy in my Debian machine.
Debian GNU/Linux 9.7 (stretch) have no xindy package.

I upgraded a machine to Debian GNU/Linux buster/sid.
Debian buster has xindy package.

Unfortunately "S" doesn't use calligraphic font in PDF file
generated from coq-8.9.0 on Debian buster.

So, I tried to generate a PDF file from coq-8.9.0 with
mathcal patch by @SkySkimmer on Debian buster.
I attach this PDF file:

[CoqRefMan-890-SkySkimmer-mathcal.pdf](https://github.com/coq/coq/files/2815502/CoqRefMan-890-SkySkimmer-mathcal.pdf)
